### ISSUE TITLE
Explain Date & Decimal handling in the docs

### ISSUE THREAD

**Issue by jakob** [jakob/Postico#476]
We should add a section to the docs that explains how date formats, time zones, and number formats are handled in Postico.

**Comment by aazaff** [comment]
Yes, please. The dates show up great in postico, but not when I export them and I don't know what transformation you are doing behind the scenes to convert.
